### DISCUSSION TITLE
blog/retrospect/5-daangn-interview

### DISCUSSION THREAD

**Discussion by giscus** [pySoo/Next-blog#77] [General]
# blog/retrospect/5-daangn-interview

4개월 만에 1차 탈락에서 최종 면접까지 갈 수 있었던 과정을 돌아보는 내용을 기록하였습니다.

https://www.enjoydev.life/blog/retrospect/5-daangn-interview

**Comment by dhdbtkd** [comment]
저도 곧 1차 화상 인터뷰 앞두고 있는데 따끈따끈한 수현님의 경험 잘봤습니다! 감사합니다

**Reply by pySoo** [reply]
안녕하세요! 끝까지 잘 준비하셔서 좋은 결과 있으시길 바라겠습니다 ☺️

**Reply by Seung-wan** [reply]
수현님 고생많으셨습니다~
면접 과정속에서 경험하셨던 것들이 글에 잘 녹아있어서 좋네요!
또한 엄청 성장하시는 계기가 될 것 같습니다!

**Reply by Seung-wan** [reply]
에고 대댓으로 잘못달았네요 ㅠㅠ

**Reply by pySoo** [reply]
@Seung-wan 헉 승완님 오랜만이에요! 긴 글 읽어주셔서 감사합니다 ㅎㅎ 🥺
이번 성장을 계기로 다시 또 여러 기회에 도전해봐야겠습니다 💪💪

**Reply by dhdbtkd** [reply]
수현님의 지난 8월처럼 화상 인터뷰에서 광탈하고 말았네요 :(ㅋㅋㅋㅋ
저 역시 이번 경험으로 부족한 부분 많이 느꼈습니다.. 잘 복기해서 탄탄한 엔지니어가 돼야겠네요.
감사합니다.

**Reply by pySoo** [reply]
@dhdbtkd 당근 인터뷰가 정말 쉽지 않죠.... 😭
그래도 서류가 합격되셨다는건 가능성이 충분하시다는 것이니까
아쉬웠던 점만 잘 복기하시면 앞으로 더 좋은 기회들 잡으실 수 있을 겁니다!! 고생 많으셨어요!

**Comment by ksjdev** [comment]
데브코스 진행하면서 면접을 같이 진행하셨다니.. 정말 고생 많으셨습니다 수현님!
비록 결과는 너무 아쉽게도 되었지만 수현님의 면접 복기 과정을 보니 한 층 더 성장하시고 나아가실 수 있을거라고 응원드리고 싶네요.
저 또한 수현님 포스팅 보면서 많은 배움을 얻고 갑니다. 정말 고생많으셨어요..!!

**Reply by pySoo** [reply]
에고 석주님 응원 감사드려요..!!!
사실 석주님 23년 회고글 읽으면서 많은 자극과 영감을 받았었는데 (특히 TIL 리스트 정말 멋지십니다..)
제 포스팅이 석주님께 도움이 되었다니 정말 기쁘네요..! 응원도 너무 감사합니다 ㅎㅎ 🙇‍♀️

**Comment by SeungHyune** [comment]
수현님 프로젝트 병행하면서 면접 준비하시기 힘드셨을 텐데 그동안 고생 많으셨습니다!! 😀
이번 면접을 통해 얻은 경험이 수현님을 더 좋은 개발자로 만들어주는 밑거름이 될 거라고 생각해요 ㅎㅎ
서류도 합격하기 힘든 당근을 최종까지 가셨다는 건 기술적으로는 더할 나위 없다는 거니까 자신감을 가지고 수현님이 어떤 개발자인지 잘 전달한다면 다음에는 좋은 소식이 있을 거라고 
생각합니다!!

저는 수현님과 대화해 보면서 함께 일하고 싶은 개발자란 이런 분이구나를 많이 느꼈습니다!! (저도 누구에게나 함께 일하고 싶은 개발자가 되기 위해 노력 중입니다 ㅎㅎ)
지금도 폭풍 성장하고 계시지만 앞으로가 더 기대 되기 때문에 페이스 조절 잘 하셔서 성장하셨으면 좋겠습니다 ㅎㅎ (저도 뒤따라 가겠습니다  😇)

**Reply by pySoo** [reply]
승현님 말씀을 어쩜 이렇게 따뜻하게 해주시나요! 힘되는 말씀들 너무 감사합니다 🙇‍♀️
저한테 그런 인상을 받으셨다니 정말 기쁘네요 ㅎㅎ 🥺

저는 승현님 프로젝트 작업물들을 보면서 3개월 전과 비교했을 때 (저희 VanillaJS로 이것저것 만들던 시절 ㅎㅎ)
엄청난 속도로 성장하시는게 느껴져서 저도 더 열심히 해야겠다는 좋은 자극과 동기부여를 받았습니다!

그리고 저희 동욱팀 다들 열정 넘치시고 커뮤니케이션 능력도 훌륭하셔서
저또한 저희 팀원분들의 미래가 정말 기대되는 것 같습니다 ㅎㅎ 저희 사이드 프로젝트도 멋있게 완성해봐요! 블로그도 찾아주셔서 감사합니다 👍

**Comment by Suzzzzzy** [comment]
저는 화상면접에서 부터 너무 긴장했던 기억이 있습니다.. 그래서 결국 화상면접에서 떨어졌었구요ㅠㅜ
단기간에 스스로를 발전시키고 좋은 결과를 얻어낸 것이 정말 멋있습니다!
좋은 글 덕분에 저도 자극 받고 갑니다! 앞으로를 더 응원하겠습니다!

**Reply by pySoo** [reply]
안녕하세요 수지님! 응원의 말씀 너무 감사드립니다 ☺️
저도 긴장을 많이 하는 편이라서 첫 면접 때는 수지님과 같은 상황이었습니다.. 😭

도움이 될지 모르겠지만, 제가 긴장을 극복하기 위해 시도했던 방법들을 조금 공유 드리자면..!
지금 이 면접이 평가받는 자리가 아니고 같이 일하게 될 동료들과 얘기를 나누는 자리라고 생각하면 조금은 더 편하게 제 생각을 말할 수 있더라구요!
그리고 머리가 백지 상태가 될 때는 면접관님께 다시 질문을 요청드리거나 시간을 주실 수 있는지 여쭤보면서 생각을 정리하는 편이에요

사실 무엇보다 가장 좋은 방법은 면접을 많이 경험해보는 것 같습니다.. ㅎㅎ
이번 경험을 통해 공짜 오답노트 생겼다고 생각하시구 아쉬웠던 점들 잘 복기하시면
분명 앞으로 좋은 결과 있으시리라고 생각해요! 저도 응원하고 있겠습니다 💪
